src/types/DurationEventHandler.ts

```diff
@@ -0,0 +1,32 @@
+import { EventInterface, EventHandler } from './Event';
+
+export enum DurationEventsPhase {
+  BEGIN = 'B',
+  END = 'E',
+}
+
+export interface DurationEventInterface<T, K> extends EventInterface {
+  args: Map<T, K>;
+}
+
+export class DurationEventHandler extends EventHandler {
+  constructor(public event: DurationEventInterface<string, number>) {
+    super(event);
+    this.event.ph = this.event.ph as DurationEventsPhase;
+  }
+  process(): DurationEventInterface<string, number> {
+    const args = new Map();
```

Comment:
Is there a reason for using ES `Map` and not plain JavaScript objects?

Reply:
No reason in particular, I think objects would be a better choice here, because the args are highly nested in the case of Chrome Dev Profiles. 

Will change this.